### ISSUE TITLE
Path Support in cypher

### ISSUE THREAD

**Issue by maguec** [RedisGraph/RedisGraph#272]
As a user I would like to query and receive all hops along the way:

Example relationship 
```
Node --subnet--> Subnet --subnet--> VPC
```

Example Query:

```
MATCH p=(n:Node{node_id:'i-21122112'})-[r:subnet*1..2]->() RETURN p 
```

Expected Response (includes all hops along the way):

```
"p"

[{"public_ip":"1.2.3.4", "ip":"10.2.0.5","node_id":"i-21122112"} ,{},{"name":"subnet-1east-1a","cidr":"10.252.0.0/24"}]
,
[{"public_ip":"1.2.3.4", "ip":"10.2.0.5","node_id":"i-21122112"} ,{},{"name":"subnet-east-1a", "cidr":"10.252.0.0/24"}{"name":"subnet-1east-1a","cidr":"10.252.0.0/24"},{},{"vpc_id
":"vpc-12345656","name":"myVPC","cidr":"10.252.0.0/16"}]
```

Actual Response:

```
(error) ERR unknown command `MATCH`, with args beginning with: 
```

**Comment by tkhader** [comment]
Hi team, there has not been any activity on this issue from long. Is it completed and not updated here or is yet to be picked ?

**Comment by K-Jo** [comment]
Hi @tkhader this will be enabled by two stories we're working on right now: 1) returning a graph response #339 2) using a different parser for our cypher queries. #318

**Comment by DvirDukhan** [comment]
closed by #708
